rust/cas_client/src/data_transport.rs

```diff
@@ -54,22 +56,26 @@ enum RetryError {
     #[error("{0}")]
     Hyper(#[from] hyper::Error),
 
+    #[error("{0}")]
+    HyperLegacy(#[from] hyper_util::client::legacy::Error),
+
     #[error("Request Error: {0}")]
     Request(#[from] anyhow::Error),
 
     /// Should only be used for non-success errors
     #[error("Status Error: {0}")]
-    Status(http::StatusCode),
+    Status(hyper::StatusCode),
 }
 fn is_status_retriable(err: &RetryError) -> bool {
     match err {
         RetryError::Hyper(_) => true,
+        RetryError::HyperLegacy(_) => true,
```

Comment:
are you sure that tls errors should be marked retryable?

Reply:
the HyperLegacy errors are not TLS errors, rather when the client makes a request and making the request failed. This replaces a subset of what used to be under the RetryError::Hyper(_) errors.

Reply:
ack